source/_components/sensor.wunderground.markdown

```diff
@@ -136,25 +136,27 @@ Configuration variables:
 
 All the conditions listed above will be updated every 5 minutes.
 
-<p class='note'>
-    <a name="12h">[12h]</a> - For 12 hour forecasts, you could replace
-    the ```_1d_``` part of the sensor name to get forecast for different
-    period/daytime, e.g. ```weather_2n``` will give you forecast for
-    tomorrow night. Valid values for day are ```1``` to ```4``` and
-    valid values for daytime are ```d``` or ```n```.
-</p>
+### 12 hour forecasts
 
-<p class='note'>
-    <a name="1d">[1d]</a> - For daily forecasts, you could replace the number
-    in ```_1d_``` part of the sensor name only. Valid values are from
-    ```1``` to ```4```.  That will give you forecast for different day.
-</p>
+Monitored conditions marked with <a name="12h">[12h]</a> are 12 hour 
+forecasts. To get a forecast for different period/daytime replace the
+`_1d_` part of the sensor name.  e.g. `weather_2n` will give you forecast for
```

Comment:
This is on purpose, in fact what I call 12h forecast here is called daily forecast by the WU API, the problem is that for some conditions they do provide more fine grained granularity and provide values for day and night, and for some they don't, so I just wanted to make the sensor naming a bit more uniform and let users use the same convention, unless they really do care about what happens outside during nighttime.